benchmarks/multiplatform/benchmarks/src/desktopMain/kotlin/main.desktop.kt

```diff
@@ -8,5 +8,11 @@ import kotlinx.coroutines.runBlocking
 
 fun main(args: Array<String>) {
     Config.setGlobalFromArgs(args)
-    runBlocking(Dispatchers.Main) { runBenchmarks() }
+
+    if (Config.runServer) {
+        // Start the benchmark server to receive results from browsers
+        BenchmarksSaveServer.start()
```

Comment:
what do you think about creating a separate module for a server? So the `benchmarks` module won't be responsible for anything else than running the benchmarks.

Then the `runBrowserAndSaveStats` would run the server task on a different module:

For example
```
serverProcess = ProcessBuilder("./gradlew", "startCollectorServer:run", "-PrunArguments=runServer=true saveStatsToJSON=true")
```